integration_test/gce/gce_testing.go

```diff
@@ -16,69 +16,64 @@
 
 /*
 Package gce holds various helpers for testing the agents on GCE.
+
 To run a test based on this library, you can either:
 
 * use Kokoro by triggering automated presubmits on your change, or
 * use "go test" directly, after performing the setup steps described
-in README.md.
+  in README.md.
```

Comment:
These indents area erroneous (https://tip.golang.org/doc/comment#mistakes). Let's undo them? Or is this a merge issue?

Reply:
bad merge